tests/slow/src/main/scala/tests/scalacli/BaseScalaCliSuite.scala

```diff
@@ -0,0 +1,166 @@
+package tests.scalacli
+
+import java.io.File
+import java.nio.file.Files
+import java.util.concurrent.Executors
+import java.util.concurrent.TimeoutException
+
+import scala.concurrent.Future
+import scala.concurrent.Promise
+import scala.concurrent.duration._
+
+import scala.meta.internal.metals.BuildInfo
+import scala.meta.internal.metals.Messages
+import scala.meta.internal.metals.scalacli.ScalaCli
+
+import ch.epfl.scala.bsp4j.MessageType
+import org.eclipse.lsp4j.InitializeResult
+import org.eclipse.lsp4j.MessageActionItem
+import tests.BaseLspSuite
+import tests.ScriptsAssertions
+
+abstract class BaseScalaCliSuite(protected val scalaVersion: String)
```

Comment:
Ach and also I moved the tests from BaseScalaCLISuite as they were just run once anyway,